build/release/release.sh

```diff
@@ -14,6 +14,7 @@ case ${action} in
         ;;
 
     publish.init)
+        publish_version_file
```

Comment:
uploading binaries to S3 happens as part of publish. I think we need to upload the version file too. Not sure I see the side effects.